en/docs/reference/connectors/amazons3-connector/amazons3-connector-example.md

```diff
@@ -204,47 +183,62 @@ We can use Curl or Postman to try the API. The testing steps are provided for cu
 
 ### Creating a bucket in Amazon S3
 
-1. Create a file called data.xml with the following content. Note that the bucket region is `us-east-2`. If you need to create the bucket in a different region, modify the hard coded region of the API configuration accordingly.
-    ```xml
-    <createBucket>
-        <bucketName>wso2engineers</bucketName>
-        <bucketRegion>us-east-2</bucketRegion>
-    </createBucket>
+1. Create a file called `data.json` with the following content. Note that the bucket region is `us-east-2`. If you need to create the bucket in a different region, modify the hard coded region of the API configuration accordingly.
```

Comment:
The us-east-2 region config isn't there in the json sample below. Can we modify this?

Reply:
As mentioned, the us-east-2 is hardcoded in the API, therefore we don't need to send it in the JSON payload.